### ISSUE TITLE
Using ngModel with OnPush change detection results in switch state

### ISSUE THREAD

**Issue by petester42** [webcat12345/ngx-ui-switch#243]
When loading the switch for the first time in a parent that is set to OnPush change detection, the value is not retained. The problem does not happen when using `[checked]` but does occur when using `[ngModel]`. 

I have illustrated the issue [here](https://stackblitz.com/edit/angular-lacnhn?embed=1&file=app/app.component.html)

**Comment by cmckni3** [comment]
Thanks for the stackblitz reproduction! Will look into it shortly.

**Comment by cmckni3** [comment]
I see the issue. It has to do with Angular's change detection. I am working on a fix.

For now, you would have to call `markForCheck` manually.

```typescript
  constructor(private cdr: ChangeDetectorRef) {}

  ngAfterViewInit() {
    this.cdr.markForCheck();
  }
```

**Comment by cmckni3** [comment]
Fixed in 2.0.1

**Comment by petester42** [comment]
Thanks 👍
